LadybugTools_Adapter/Convert/FromBHoM.cs

```diff
@@ -0,0 +1,79 @@
+﻿using BH.oM.LadybugTools;
+using System;
+using BH.Engine.Serialiser;
+using System.Collections.Generic;
+using System.Text;
+using BH.oM.Base;
+using System.IO;
+using BH.oM.Adapter;
+using BH.Engine.Adapter;
+using BH.oM.Base.Debugging;
+
+namespace BH.Adapter.LadybugTools
+{
+    public static partial class Convert
+    {
+        public static bool FromBHoM(this ILadybugTools input, FileSettings file)
+        {
+            string json = ICustomify(input);
+            File.WriteAllText(file.GetFullFileName(), json);
+            List<Event> events = BH.Engine.Base.Query.CurrentEvents();
+            return events.Count == 0;
+        }
+
+        public static string ICustomify(this ILadybugTools LBTObject)
+        {
+            if (LBTObject == null)
+            {
+                BH.Engine.Base.Compute.RecordError("Input object is null.");
```

Comment:
Update this error message to give more information to the user on what was trying to be done so they can best rectify the issue.